data/pathfinder/paizo/campaign_setting/inner_sea_combat/isc_abilities_class.lst

```diff
@@ -580,7 +581,6 @@ CATEGORY=Archetype|Barbarian Archetype ~ Jungle Rager.MOD
 CATEGORY=Archetype|Rogue Archetype ~ Kintargo Rebel.MOD																														PREMULT:1,[PREABILITY:1,CATEGORY=Archetype,Rogue Archetype ~ Kintargo Rebel],[!PREFACT:1,ABILITIES,Rogue_Archetype_TrapSense=True,Rogue_Archetype_UncannyDodge=True]																																																																																																																																																																																																												FACT:Rogue_CF_TrapSense|true	FACT:Rogue_CF_UncannyDodge|true	FACT:Rogue_Archetype_TrapSense|true	FACT:Rogue_Archetype_UncannyDodge|true
 CATEGORY=Archetype|Monk Archetype ~ Monk of the Seven Forms.MOD																													PREMULT:1,[PREABILITY:1,CATEGORY=Archetype,Monk Archetype ~ Monk of the Seven Forms],[!PREFACT:1,ABILITIES,Monk_Archetype_StunningFist=True,Monk_Archetype_ManeuverTraining=True,Monk_Archetype_AbundantStep=True,Monk_Archetype_QuiveringPalm=True]																																																																																																																																																																																															FACT:Monk_CF_StunningFist|true	FACT:Monk_CF_ManeuverTraining|true	FACT:Monk_CF_AbundantStep|true	FACT:Monk_CF_QuiveringPalm|true	FACT:Monk_Archetype_StunningFist|true	FACT:Monk_Archetype_ManeuverTraining|true	FACT:Monk_Archetype_AbundantStep|true	FACT:Monk_Archetype_QuiveringPalm|true
 CATEGORY=Archetype|Rogue Archetype ~ Oheno Liberator.MOD																														PREMULT:1,[PREABILITY:1,CATEGORY=Archetype,Rogue Archetype ~ Oheno Liberator],[!PREFACT:1,ABILITIES,Rogue_Archetype_Trapfinding=True,Rogue_Archetype_TrapSense=True,Rogue_Archetype_UncannyDodge=True]																																																																																																																																																																																																						FACT:Rogue_CF_Trapfinding|true	FACT:Rogue_CF_TrapSense|true	FACT:Rogue_CF_UncannyDodge|true	FACT:Rogue_Archetype_Trapfinding|true	FACT:Rogue_Archetype_TrapSense|true	FACT:Rogue_Archetype_UncannyDodge|true
-CATEGORY=Archetype|Monk Archetype ~ Ouat.MOD																																PREMULT:1,[PREABILITY:1,CATEGORY=Archetype,Monk Archetype ~ Ouat],[!PREFACT:1,ABILITIES,Monk_Archetype_WholenessOfBody=True,Dwarf_Trait_DefensiveTraining=True,Dwarf_Trait_Stonecunning=True,Dwarf_Trait_WeaponFamiliarity=True]																																																																																																																																																																																																		FACT:Monk_CF_WholenessOfBody|true	FACT:Monk_Archetype_WholenessOfBody|true	FACT:Dwarf_Trait_DefensiveTraining|true	FACT:Dwarf_Trait_Stonecunning|true	FACT:Dwarf_Trait_WeaponFamiliarity|true
```

Comment:
Again, removing an entire thing, but leaving the rest unchanged seems odd.

Reply:
The entire archetype was rebuilt. This section is no longer needed, since it was the MOD patch to make it work with the FACT system.